klasses/models.py

```diff
@@ -41,6 +41,14 @@ class Bootcamp(models.Model):
 
     title = models.TextField()
     legacy = models.BooleanField(default=False)
+    readable_id = models.CharField(
+        null=True,
+        blank=True,
```

Comment:
readable_id in other apps is not nullable. Is there a reason we are not forcing it?

One reason I think of is that we don't have data for all of these now. If Yes, I think we should do the following:
1. Create migration for this as is nullable (This is already done in this PR 0029)
2. Create another migration making this non-nullable 
3. This can very well be the migration 0030 that you created but it will now do two tasks in order a) Backfill with appropriate values b) Alter field and make non-nullable ([This is an example from xPRO](https://github.com/mitodl/mitxpro/pull/2786/files#diff-75e20b31ad6576874eb9bee3b7edec5a3a1ed36acc86e408e56093016bd13129))

Reply:
If you would like to do this in a followup PR we would end up with 3 migrations instead of doing it in 2, But I don't have issues with that as well. Just let me know.

Reply:
Yeah, That sounds good.

Reply:
I reverted 0030 migration, the backfill and make it required field will be addressed in https://github.com/mitodl/hq/issues/3975